File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/consts/ErdConst.java
package org.javamaster.invocationlab.admin.consts;

import java.util.concurrent.atomic.AtomicInteger;

/**
 * @author yudong
 */
public class ErdConst {
    public static final String ADMIN = "admin";
    public static final String ADMIN_CODE = "100001";
    public static final String ERD_PREFIX = ADMIN + ":erd:";
    // cookie中不能包含:等特殊字符,否则报错
    public static final String COOKIE_TOKEN = ADMIN + "-erd-token";
    public static final String SUB_ROLE_SUFFIX = ":ROLE_IDS";

    public static final String PROJECT_DS = ERD_PREFIX + "project:ds";
    public static final String PROJECT_STATISTIC = ERD_PREFIX + "statistic";
    public static final String PROJECT_ROLE_GROUP_USERS = ERD_PREFIX + "project:roleGroup:user:";
    public static final String USER_PROJECT_ROLE_GROUP = ERD_PREFIX + "userRoleGroup:";
    public static final String PROJECT_ROLE_GROUP_SUB_ROLES = ERD_PREFIX + "roleGroup:subRole:";

    public static final String PROJECT_QUERY_TREE = ERD_PREFIX + "tree";
    public static final String QUERY_TREE_TREE_NODE = ERD_PREFIX + "treeNode";

    public static final String ANGEL_PRO_ALLOW = ERD_PREFIX + "angelProAllow";
    public static final String ANGEL_PRO_RPC_ALLOW = ERD_PREFIX + "angelProRpcAllow";

    public static final String QUERY_RES = ERD_PREFIX + "queryRes";
    public static final String EXEC_COLLECTION_NAME = ERD_PREFIX + "queryTableName";
    public static final String MONGO_KEY_NAME = "_id";

    public static final String NULL_VALUE = "<null>";
    public static final String ROW_OPERATION_TYPE = "rowOperationType";
    public static final String KEY = "key";
    public static final String INDEX = "index";
    public static final AtomicInteger COUNTER = new AtomicInteger(0);
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/consts/ErdConst.kt
package org.javamaster.invocationlab.admin.consts

/**
 * @author yudong
 */
object ErdConst {
    private const val MH_ERD: String = "mh-erd"
    const val ADMIN_CODE: String = "1000001"
    const val ERD_PREFIX: String = "$MH_ERD:erd:"

    // cookie中不能包含:等特殊字符,否则报错
    const val COOKIE_TOKEN: String = "$MH_ERD-erd-token"
    const val SUB_ROLE_SUFFIX: String = ":ROLE_IDS"

    const val PROJECT_DS: String = ERD_PREFIX + "project:ds"
    const val PROJECT_STATISTIC: String = ERD_PREFIX + "statistic"
    const val PROJECT_ROLE_GROUP_USERS: String = ERD_PREFIX + "project:roleGroup:user:"
    const val USER_PROJECT_ROLE_GROUP: String = ERD_PREFIX + "userRoleGroup:"
    const val PROJECT_ROLE_GROUP_SUB_ROLES: String = ERD_PREFIX + "roleGroup:subRole:"

    const val PROJECT_QUERY_TREE: String = ERD_PREFIX + "tree"
    const val QUERY_TREE_TREE_NODE: String = ERD_PREFIX + "treeNode"

    const val ANGEL_PRO_ALLOW: String = ERD_PREFIX + "angelProAllow"
    const val ANGEL_PRO_RPC_ALLOW: String = ERD_PREFIX + "angelProRpcAllow"

    const val QUERY_RES: String = ERD_PREFIX + "queryRes"
}
